apis/Google.Cloud.Logging.Console/docs/index.md

```diff
@@ -31,6 +31,34 @@ var builder = WebApplication.CreateBuilder(args);
 builder.Logging.AddGoogleCloudConsole(options => options.IncludeScopes = true);
 ```
 
+## Log trace correlation
+
+The console formatter supports Google Cloud Trace and log correlation. This means that
+if the application using the console formatter is running in Google Cloud and exports traces
```

Comment:
Or if it's running anywhere else and it's somehow exporting traces to Google Cloud Trace.